jbpm-services/jbpm-executor-cdi/src/test/filtered-resources/META-INF/persistence.xml

```diff
@@ -10,6 +10,8 @@
     <mapping-file>META-INF/Executor-orm.xml</mapping-file>
     <class>org.jbpm.executor.entities.ErrorInfo</class>
     <class>org.jbpm.executor.entities.RequestInfo</class>
+    <class>org.jbpm.process.audit.ProcessInstanceLog</class>
+    <class>org.jbpm.runtime.manager.impl.jpa.ExecutionErrorInfo</class>
```

Comment:
why is this here if the test is in test coverage ?

Reply:
The entities need to be mapped, otherwise they can't be found when we run the query.
I didn't get what you mean with:
> ... if the test is in test coverage

Reply:
Hi, 
This should be consider a smell because you are mapping an entity that does not belong to this package/module here. This should tip you that you are breaking module coherence.
I understand because you are modifying the persistence here it needs process  instance log (not defined here) and Execution error info (not defined here either). Same comment as below.

Reply:
Got your point now. Nice catch.
Thanks.

Reply:
@elguardian wouldn't the above mappings also be a smell? They're from another package as well.

````xml
    <class>org.jbpm.executor.entities.ErrorInfo</class>
    <class>org.jbpm.executor.entities.RequestInfo</class>
````